browser/BUILD.gn

```diff
@@ -152,7 +145,6 @@ source_set("browser_process") {
     "//brave/components/speedreader:buildflags",
     "//brave/components/weekly_storage",
     "//brave/services/network/public/cpp",
-    "//chrome/common",
```

Comment:
are you sure nothing else needs this? One of the problems with these giant targets is that it's hard to know which deps go with which files. Ideally `component_updater`, `geolocation`, `metrics` and `search_engines` would go in their own BUILD.gn files located in those subdirectories

Reply:
Right, this should be kept here. I was too hurry in order to fix this regression before starting my vacation. Fixed.